src/main/java/org/spongepowered/common/service/scheduler/SchedulerBase.java

```diff
@@ -52,14 +52,14 @@ protected String nextName(PluginContainer plugin) {
 
     /**
      * Gets the timestamp to update the timestamp of a task. This method is task
-     * sensitive to support different timestamp types i.e. wall clock and ticks.
+     * sensitive to support different timestamp types i.e. nanotime and ticks.
      *
      * @param task The task
      * @return Timestamp for the task
      */
     protected long getTimestamp(ScheduledTask task) {
-        // Supports wall clock time by default
-        return System.currentTimeMillis();
+        // Supports nanotime by default
```

Comment:
@simon816 The issue is that `System.nanoTime()` is explicitly not tied to any system or wall clock, it is completely arbitrary and only useful for "x time has passed" checks. Leaving the comment would be misleading.

As specified in the javadocs:
> This method can only be used to measure elapsed time and is not related to any other notion of system or wall-clock time. 
<..>
> The values returned by this method become meaningful only when the difference between two such values, obtained within the same instance of a Java virtual machine, is computed.